extensions/azdata/package.json

```diff
@@ -20,17 +20,66 @@
   },
   "main": "./out/extension",
   "contributes": {
-    "configuration":  {
-      "type": "object",
-      "title": "%azdata.config.title%",
-      "properties": {
-        "azdata.logDebugInfo": {
-          "type": "boolean",
-          "default": false,
-          "description": "%azdata.config.debug%"
+    "configuration": [
+      {
+        "type": "object",
+        "title": "%azdata.config.title%",
+        "properties": {
+          "azdata.logDebugInfo": {
+            "type": "boolean",
+            "default": false,
+            "description": "%azdata.config.debug%"
+          },
+          "azdata.acceptEula": {
+            "type": "string",
+            "default": "Ask Later",
```

Comment:
Values for this are typically camel-cased since they aren't localized. You should use enumDescriptions to give the localized text descriptions for each of these values. See `enum / enumDescriptions` in https://code.visualstudio.com/api/references/contribution-points#contributes.configuration